oak_restricted_kernel_sdk/src/dice.rs

```diff
@@ -15,76 +15,36 @@
 //
 
 use crate::syscall::read;
-use oak_crypto::{
-    encryptor::EncryptionKeyProvider,
-    hpke::{Deserializable, PrivateKey, PublicKey},
-};
-use oak_dice::{
-    cert::{
-        cose_key_to_hpke_public_key, get_claims_set_from_certificate_bytes,
-        get_public_key_from_claims_set,
-    },
-    evidence::{RestrictedKernelDiceData, P256_PRIVATE_KEY_SIZE, X25519_PRIVATE_KEY_SIZE},
-};
-use oak_remote_attestation::proto::oak::attestation::v1::Evidence;
+use anyhow::Ok;
+use oak_crypto::encryptor::EncryptionKeyProvider;
+use oak_dice::evidence::{Evidence, RestrictedKernelDiceData, P256_PRIVATE_KEY_SIZE};
 use oak_restricted_kernel_interface::DICE_DATA_FD;
 use p256::ecdsa::SigningKey;
 use zerocopy::{AsBytes, FromZeroes};
 
 /// Wrapper for DICE evidence and application private keys.
-pub struct DiceWrapper {
+#[allow(dead_code)]
+struct DiceWrapper {
     pub evidence: Evidence,
     pub encryption_key: EncryptionKeyProvider,
     pub signer: Signer,
 }
 
+#[allow(dead_code)]
 impl DiceWrapper {
     pub fn try_create() -> anyhow::Result<Self> {
```

Comment:
this will be used in the next PR (that shares this data using a `OnceCell`, so allowing it here for now.